runtime/src/snapshot_utils.rs

```diff
@@ -373,11 +373,20 @@ pub fn archive_snapshot_package(
         timer.as_ms(),
         metadata.len()
     );
+
     datapoint_info!(
-        "snapshot-package",
+        "archive-snapshot-package",
         ("slot", snapshot_package.slot(), i64),
         ("duration_ms", timer.as_ms(), i64),
```

Comment:
I could also prefix this "duration" field too, but I personally didn't need the distinction yet. Can always be added later.

Reply:
could also make the datapoint name conditional on the kind of snapshot, but now I'm bikeshedding